changelogs/unreleased/gh-8744-fix-chunked-with-content-length.md

```diff
@@ -0,0 +1,4 @@
+## bugfix/lua/http client
+
+* Fixed "Transfer-Encoding: chunked" setting even if "Content-Length" header
+  exists for stream requests.
```

Comment:
Please, add an issue number (check other changelog entries for the format).

Reply:
Done:

`* Fixed "Transfer-Encoding: chunked" setting even if "Content-Length" header
  exists for stream requests (gh-8744).`